Makefile

```diff
@@ -57,6 +57,7 @@ endif
 .PHONY: server
 server:
 ifneq ($(HAS_SERVER),)
+	$(info Are you sure you want to build the plugin with an id of "com.mattermost.plugin-starter-template"? Consider editing plugin.json to configure your project.)
```

Comment:
This will show the message always, right? I mean... even if I have edited my plugin.json and have the correct id, I will still receive this message. Shouldn't we do a check on the manifest id somehow? Or add something more like "edit the plugin.json and remove this line".

Reply:
Hey @larkox, thanks for your review:) Instead of having a check, we can do something simple like adding **"Ignore if already edited"** to the end of the message. How does that sound? 
/cc @hanzei

Reply:
I don't like it, because a warning that always appear is a warning you ignore. And the problem will still be there.

Reply:
What change would you like to suggest here?

Reply:
Something like:
```
ifeq ($(PLUGIN_ID), com.mattermost.plugin-starter-template)
    echo "Are you sure you want to build the plugin with an id of "com.mattermost.plugin-starter-template"? Consider editing plugin.json to configure your project."
endif
```

I would even consider exiting and not allowing the plugin to be compiled, but that may be too drastic :P